### PR TITLE
Add: Grisbi version 2.0.5

### PR DESCRIPTION
**Pull request by mnival**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80832)

### CODE REVIEW

**Comment by mnival** [comment]
@microsoft-github-policy-service agree

**Comment by mnival** [comment]
@microsoft-github-policy-service agree

File: manifests/g/Grisbi/Grisbi/2.0.5/Grisbi.Grisbi.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Grisbi.Grisbi
+PackageVersion: 2.0.5
+PackageLocale: en-US
+Publisher: Grisbi
```
Comment (Trenly) [inline manifests/g/Grisbi/Grisbi/2.0.5/Grisbi.Grisbi.locale.en-US.yaml:7]:
```suggestion
Publisher: Open Source
```
This must match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — APPROVED** [review]

**Pull request merged**